Bindings/Java/Matlab/examples/OpenSimCreateTugOfWarModel.m

```diff
@@ -1,79 +1,64 @@
+%                                                                         %
 % Unless required by applicable law or agreed to in writing, software     %
 % distributed under the License is distributed on an "AS IS" BASIS,       %
 % WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
 % implied. See the License for the specific language governing            %
 % permissions and limitations under the License.                          %
 % ----------------------------------------------------------------------- %
 
-% OpenSimCreateTugOfWarModel.m - script to perform the same model building and
-% simulation tasks as the MainExample from the SDK examples
-
-% This example script creates a model similar to the TugofWar API example.
-% Two muscles are created and attached to a block. Linear controllers are
-% defined for the muscles.
-
-% Pull in the modeling classes straight from the OpenSim distribution
+%% Import OpenSim libraries
 import org.opensim.modeling.*
 
-%///////////////////////////////////////////
-%// DEFINE BODIES AND JOINTS OF THE MODEL //
-%///////////////////////////////////////////
-
-% Create a blank model
+%% Instantiate an empty model
 model = Model();
 model.setName('TugOfWar')
-
-% Convenience - create a zero value Vec3
-zeroVec3 = ArrayDouble.createVec3(0);
-
-% Set gravity as 0 since there is no ground contact model in this version
-% of the example
+% Convenience - Instantiate a zero value Vec3
+zeroVec3 = Vec3(0);
+% Set gravity as 0 since we are not concerned with ground contact
 model.setGravity(zeroVec3);
 
-% GROUND BODY
-
+%% Define Bodies and Joints in the Model
 % Get a reference to the model's ground body
 ground = model.getGround();
-
 % Add display geometry to the ground to visualize in the GUI
 ground.attachGeometry(Mesh('ground.vtp'));
 ground.attachGeometry(Mesh('anchor1.vtp'));
 ground.attachGeometry(Mesh('anchor2.vtp'));
 
-% "BLOCK" BODY
-
-% Create a block Body with associate dimensions, mass properties, and DisplayGeometry
+% Instantiate a Body with mass, inertia, and a display geometry
 block = Body();
 block.setName('Block');
 block.setMass(20);
 block.setMassCenter(zeroVec3);
-% Need to set inertia
+block.setInertia(Inertia(0.133,0.133,0.133,0,0,0));
+% Add display geometry for the block
 block.attachGeometry(Mesh('block.vtp'));
 
-% FREE JOINT
-
-% Create a new free joint with 6 degrees-of-freedom (coordinates) between the block and ground bodies
+% Instantiate a Free Joint (6 DoF) that connects the block and ground.
 blockSideLength      = 0.1;
-locationInParentVec3 = ArrayDouble.createVec3([0, blockSideLength/2, 0]);
+locationInParentVec3 = Vec3(0, blockSideLength/2, 0);
 blockToGround        = FreeJoint('blockToGround', ...
                             ground, locationInParentVec3, zeroVec3, ...
                             block, zeroVec3, zeroVec3);
 
-% Set bounds on coordinates
+% Set bounds on the 6 coordinates of the Free Joint.
```

Comment:
Briefer (though perhaps less clear):
<pre>
for i=0:2, blockToGround.upd_coordinates(i).setRange(angleRange); end
for i=3:5, blockToGround.upd_coordinates(i).setRange(positionRange); end
</pre>

Reply:
I like this suggested way of doing it, so included it.